config/kustomization.yaml

```diff
@@ -0,0 +1,23 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - namespace.yaml
+  - sprayproxy.yaml
+  - route.yaml # only for openshift deployments
+  - sprayproxy-rbac.yaml
+
+namespace: sprayproxy
```

Comment:
Since we are explicitly defining the resources within namespace. Is this namespace required here?